server/controllers/finance/cash.js

```diff
@@ -140,19 +137,20 @@ function detail(req, res, next) {
 
 /**
  * POST /cash
- * Creates a cash payment against several sales or a cautionary payment.  The
- * API also supports future offline functionality by either accepting a UUID or
- * generating it if it is not present.
+ * Creates a cash payment against one or many previous sales or a cautionary
+ * payment.  If a UUID is not provided, one is automatically generated.
  */
 function create(req, res, next) {
   'use strict';
 
-  var data = req.body.payment;
+  // alias insertion data
+  const data = req.body.payment;
```

Comment:
`const` in ECMAScript 6 does not mean that the value stays constant.  It means that the binding (the MDN calls it [the reference](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const)) should not change.  For example, see this code below:
```js
const o = { value :1 };
console.log(o);  // logs { value : 1 }

o.value = 2;
console.log(o) : // logs { value : 2 }

delete o;  // returns false
console.log(o); // logs { value : 2 }
```
Clearly, the object `o` is not constant.  In this case, to actually make the value constant, you would need to use [Object.freeze](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/freeze) as shown below:
```js
const o = Object.freeze({ value : 1 });
console.log(o);  // logs { value : 1}

o.value = 2;
console.log(o); // logs { value : 1 }

delete o;  // returns false
console.log(o); // logs { value : 1 }
```